### PR TITLE
Add  Environment config file 

### LINKED ISSUE #135: Add Environment config file
**Describe the Issue**

The issue is currently we are calling this  `FlutterConfig.get(<Env Variable>)`   everywhere where it's needed. Instead, the proposed solution is to create a `class ` and declare these as static variables.  And also add **IOS Google Map API Key**  in **Env Variables** so that developers can test this app in both Android and IOS.

Thank You :)

 
 Cc: @nb9960  @Aadeesh11  @ItsAdityaKSingh

### PR DESCRIPTION
**Pull request by adarsh-technocrat**
Fixes #135 

Describe the changes you have made in this PR -

Hello Everyone In this PR I have made a few changes related to environment variables. I have bit structure the code and also added Env variable support for IOS google Maps API Key as there will be different API keys available for  Android and IOS.

Screenshots of the changes (If any) -

Note: Please check Allow edits from maintainers. if you would like us to assist in the PR.

Cc:   @nb9960   @cfsmp3  @Aadeesh11 @ItsAdityaKSingh

### CODE REVIEW

**Comment by adarsh-technocrat** [comment]
> iOS setup is still incomplete, i.e, API key still needs to be specified in the application delegate. Can you please do the needful in this PR itself? Also, Please fix the conflicting file.

Sure @nb9960 I'll make that change  :)

File: lib/components/timer.dart [hunk 1]
```diff
@@ -35,7 +35,7 @@ class _CountdownTimerPageState extends State<CountdownTimerPage>
         CountdownTimerController(endTime: endTime, onEnd: onEnd, vsync: this);
   }
 
-  void onEnd() async {
+  void onEnd() {
```
Comment (nb9960) [inline lib/components/timer.dart:38]: You reverted the changes made in recent commits, kindly undo those

Comment (adarsh-technocrat) [reply lib/components/timer.dart]: Resolved!

**Review by nb9960 — APPROVED** [review]

**Comment by nb9960** [comment]
Thanks for the PR, merged! :tada:

**Pull request merged**